Tests/SubviewDistributionTests.swift

```diff
@@ -47,7 +47,7 @@ class SubviewDistributionTests: XCTestCase {
         let pixelRounder = PixelRounder(for: view)
         // Assert that our spacing is correct above.
         XCTAssertEqual((a.bottom |--| b.top).height, relative1Space, accuracy: pixelRounder.pixelAccuracy)
-        XCTAssertEqual((b.bottom |--| c.top).height, 2 * relative1Space, accuracy: pixelRounder.pixelAccuracy)
+        XCTAssertEqual((b.bottom |--| c.top).height / 2.0, relative1Space, accuracy: pixelRounder.pixelAccuracy)
```

Comment:
Previously we were multiplying our rounding delta. Now we're dividing the actual measured space and comparing it to the relative space within a pixel distance.